uber/configspec.ini

```diff
@@ -49,6 +49,9 @@ at_the_con = boolean(default=False)
 # and behaviors such as automated emails which are ONLY turned on afterwards.
 post_con = boolean(default=False)
 
```

Comment:
Agreed about the comment being more specific. I definitely do not want to make send_emails tied that closely to this option, though - there's plenty of cases where we want send_emails to be turned off even when offline_mode is turned off. I'm fine with letting this option also disable them, though.

What is missing from Stripe payments? I removed the option from the kiosk, do you think it should also be removed from the admin sections?